hashes/README.md

```diff
@@ -30,3 +30,90 @@ Alternatively add symlinks in your `.git/hooks` directory to any of the githooks
 
 We use a custom Rust compiler configuration conditional to guard the bench mark code. To run the
 bench marks use: `RUSTFLAGS='--cfg=bench' cargo +nightly bench`.
+
+## Algorithms
+
+We currently support the following hash algorithms
+
+### [SHA-1](https://en.wikipedia.org/wiki/SHA-1)
+
+Since 2005, SHA-1 has not been considered secure against well-funded opponents
+
+Module: `sha1`
+Digest: 160 bit (20 byte)
+
+### [SHA-2](https://en.wikipedia.org/wiki/SHA-2)
+
+Supported functions from the SHA-2 family:
+
+- SHA-256
+
+Module: `sha256`
+Digest: 256 bit (32 byte)
```

Comment:
In ce1881bbfd8029cce25799ba0668ebae43259b9b

These all look fine but a little hard to keep up-to-date. The digest length is in the name of every one of these hashes, and also we may rearrange our modules so that the `Module:` line goes out of date.